### PR TITLE
cinemat.cpp: Update controls for spaceshp, convert to C-style comments

### PR DESCRIPTION
**Pull request by einstein95**
Based on control panel pics from:
- https://twitter.com/kt2soundlab/status/1547083104447123456 (JP spaceshp panel)
- https://twitter.com/kt2soundlab/status/1547121518244573186 (EN spaceshp panel)
- https://i.imgur.com/MbKG2u1h.jpg (FR spaceshp panel)
- https://i.imgur.com/xoZ0OD1h.jpg (spacewar panel)

### CODE REVIEW

File: src/mame/cinematronics/cinemat.cpp [hunk 1]
```diff
@@ -353,38 +353,60 @@ static INPUT_PORTS_START( spacewar )
...
 	PORT_BIT( 0x0100, IP_ACTIVE_LOW, IPT_JOYSTICK_LEFT ) PORT_2WAY PORT_PLAYER(1)
-	PORT_BIT( 0x0200, IP_ACTIVE_LOW, IPT_BUTTON2 ) PORT_PLAYER(2)
+	PORT_BIT( 0x0200, IP_ACTIVE_LOW, IPT_BUTTON2 ) PORT_PLAYER(2) PORT_NAME("Forward")
 	PORT_BIT( 0x0400, IP_ACTIVE_LOW, IPT_OTHER ) PORT_NAME("Option 5") PORT_CODE(KEYCODE_5_PAD)
 	PORT_BIT( 0x0800, IP_ACTIVE_LOW, IPT_OTHER ) PORT_NAME("Option 0") PORT_CODE(KEYCODE_0_PAD)
 	PORT_BIT( 0x1000, IP_ACTIVE_LOW, IPT_JOYSTICK_RIGHT ) PORT_2WAY PORT_PLAYER(2)
 	PORT_BIT( 0x2000, IP_ACTIVE_LOW, IPT_JOYSTICK_RIGHT ) PORT_2WAY PORT_PLAYER(1)
 	PORT_BIT( 0x4000, IP_ACTIVE_LOW, IPT_JOYSTICK_LEFT ) PORT_2WAY PORT_PLAYER(2)
-	PORT_BIT( 0x8000, IP_ACTIVE_LOW, IPT_BUTTON2 ) PORT_PLAYER(1)
+	PORT_BIT( 0x8000, IP_ACTIVE_LOW, IPT_BUTTON2 ) PORT_PLAYER(1) PORT_NAME("Forward")
```
Comment (cuavas) [inline src/mame/cinematronics/cinemat.cpp:362]: It’s unhelpful to have duplicate port names in a system.

File: src/mame/cinematronics/cinemat.cpp [hunk 2]
```diff
@@ -353,38 +353,60 @@ static INPUT_PORTS_START( spacewar )
...
 	PORT_DIPSETTING(    0x01, "1:30/coin" )
 	PORT_DIPSETTING(    0x02, "2:00/coin" )
-	PORT_BIT( 0x04, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_PLAYER(2)
-	PORT_BIT( 0x08, IP_ACTIVE_LOW, IPT_BUTTON3 ) PORT_PLAYER(2)
-	PORT_BIT( 0x10, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_PLAYER(1)
-	PORT_BIT( 0x20, IP_ACTIVE_LOW, IPT_BUTTON3 ) PORT_PLAYER(1)
+	PORT_BIT( 0x04, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_PLAYER(2) PORT_NAME("Fire")
+	PORT_BIT( 0x08, IP_ACTIVE_LOW, IPT_BUTTON3 ) PORT_PLAYER(2) PORT_NAME("Hyperspace")
+	PORT_BIT( 0x10, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_PLAYER(1) PORT_NAME("Fire")
+	PORT_BIT( 0x20, IP_ACTIVE_LOW, IPT_BUTTON3 ) PORT_PLAYER(1) PORT_NAME("Hyperspace")
```
Comment (cuavas) [inline src/mame/cinematronics/cinemat.cpp:373]: Same here, this is going to be confusing in menus.  The `%p` pseudo-format for localised player exists for a reason.  But that aside, in most cases, it’s fine to just leave the default button names and allow the port name plugin to be used to supply “friendly” names.

File: src/mame/cinematronics/cinemat.cpp [hunk 3]
```diff
@@ -353,38 +353,60 @@ static INPUT_PORTS_START( spacewar )
...
 	PORT_MODIFY("SWITCHES")
 	PORT_DIPNAME( 0x03, 0x00, "Time" ) PORT_DIPLOCATION("SW1:!4,!3")
 	PORT_DIPSETTING(    0x00, "1:00/coin" )
 	PORT_DIPSETTING(    0x01, "1:30/coin" )
 	PORT_DIPSETTING(    0x02, "2:00/coin" )
 	PORT_DIPSETTING(    0x03, "2:30/coin" )
+	PORT_BIT( 0x04, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_PLAYER(2) PORT_NAME("Fire Missile")
+	PORT_BIT( 0x08, IP_ACTIVE_LOW, IPT_BUTTON2 ) PORT_PLAYER(2) PORT_NAME("Hyperspace")
+	PORT_BIT( 0x10, IP_ACTIVE_LOW, IPT_BUTTON1 ) PORT_PLAYER(1) PORT_NAME("Fire Missile")
+	PORT_BIT( 0x20, IP_ACTIVE_LOW, IPT_BUTTON2 ) PORT_PLAYER(1) PORT_NAME("Hyperspace")
```
Comment (cuavas) [inline src/mame/cinematronics/cinemat.cpp:409]: You’ve made duplicates here, too.

**Review by cuavas — CHANGES_REQUESTED** [review]

File: src/mame/cinematronics/cinemat.cpp (hunk 2 above)
Comment (einstein95) [reply src/mame/cinematronics/cinemat.cpp:373]: Ah, I wasn't aware of the portname plugin. I'll remove the added labels.

Comment (happppp) [reply src/mame/cinematronics/cinemat.cpp]: Personally, I'm ok with default PORT_NAME. I wouldn't use it when a game only has 2 buttons like Attack/Jump, but I would add them if things get confusing. For example: https://commons.wikimedia.org/wiki/File:Basketball_2_by_Mattel_Electronics,_Made_In_Hong_Kong,_Copyright_1979_%28LED_Electronic_Handheld_Game%29.jpg
Or this one with 24 buttons: https://www.schach-computer.info/wiki/images/d/d1/Novag_Diablo-1.jpg

The portname plugin is nice, but hard to maintain due to separate code/data (eg. port tags, bitmasks). Plus expecting users to find/download/update the portname files.

**Pull request merged**